src/clojure/duratom/utils.clj

```diff
@@ -202,16 +202,21 @@
                                                         [[:id :int] [:value col-type]]))
     (catch BatchUpdateException _ '(0)))) ;; table already exists!
 
-(defn get-pgsql-value [db table-name row-id read-it!]
```

Comment:
I'm not sure about `get-sql-value` as it implies that it will work with any SQL dialect, which is not the case (e.g. OracleSQL doesn't support `LIMIT` - it does it differently). However, I will admit that the name could be improved - how about `pgsql-current` or something along these lines?

Reply:
Ok, I just figured it was a bit silly to have two different functions that work the same for both postgred and sqlite. I do know that different implementations require different syntax.

Reply:
To be reverted...